<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchLine">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Искать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fileSearchButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Открыть файл</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ColoredQTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="id_label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="id_input"/>
      </item>
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>ФИО:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_input"/>
      </item>
      <item>
       <widget class="QLabel" name="year_label">
        <property name="text">
         <string>Год рождения:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="year_input"/>
      </item>
      <item>
       <widget class="QLabel" name="sex_label">
        <property name="text">
         <string>Пол:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sex_input">
        <item>
         <property name="text">
          <string>Мужской</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Женский</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_employee_btn">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColoredQTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>coloredqtextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sFindEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_employee_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sAddEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>searchLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_employee_btn</sender>
   <signal>clicked()</signal>
   <receiver>id_input</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_employee_btn</sender>
   <signal>clicked()</signal>
   <receiver>name_input</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_employee_btn</sender>
   <signal>clicked()</signal>
   <receiver>year_input</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSearchButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sSearchFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sFindEmployee()</slot>
  <slot>sAddEmployee()</slot>
  <slot>sSearchFile()</slot>
 </slots>
</ui>
